### ISSUE TITLE
AttributeError: module 'numpy' has no attribute 'str'.

### ISSUE THREAD

**Issue by klshen8386** [scverse/scvi-tools#2851]
seeing AttributeError: module 'numpy' has no attribute 'str'. when running code below (loading pbmc dataset)

```python
adata = scvi.data.pbmc_dataset()
```

```pytb
AttributeError: module 'numpy' has no attribute 'str'.
`np.str` was a deprecated alias for the builtin `str`. To avoid this error in existing code, use `str` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.str_` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

#### Versions:

> 0.20.3

**Comment by martinkim0** [comment]
Hi, does the error replicate with scvi-tools >= 1.0.0? If you need 0.20.3 for reproducibility purposes, I would downgrade numpy to 1.20 or below.

**Comment by klshen8386** [comment]
Hello, for dependency reasons I require scvi-tools < 1.0.0 > 0.16.0, however when I attempt to downgrade numpy below 1.20 (in this case, 1.19), I am receiving the error RuntimeError: Running cythonize failed! error: metadata-generation-failed

**Comment by martinkim0** [comment]
Can you try numpy 1.23 instead? I believe np.str was removed in 1.24.

**Comment by klshen8386** [comment]
Issue fixed - numpy 1.23 was compatible with scvi-tools 0.20.3. Thank you!

**Comment by klshen8386** [comment]
Issue fixed - numpy 1.23 was compatible with scvi-tools 0.20.3. Thank you!
